cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/SvInjectionConversion.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.elements;
+
+import com.powsybl.cgmes.conversion.Context;
+import com.powsybl.cgmes.model.CgmesTerminal;
+import com.powsybl.iidm.network.*;
+import com.powsybl.triplestore.api.PropertyBag;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public class SvInjectionConversion extends AbstractIdentifiedObjectConversion {
+
+    public SvInjectionConversion(PropertyBag p, Context context) {
+        super("SvInjection", p, context);
+        String tn = p.getId("TopologicalNode");
+        Terminal associatedTerminal = context.terminalMapping().findFromTopologicalNode(tn);
+        if (associatedTerminal == null) {
+            cgmesTerminal = context.cgmes().terminal(context.terminalMapping().findCgmesTerminalFromTopologicalNode(tn));
+            if (cgmesTerminal == null || context.cgmes().voltageLevel(cgmesTerminal) == null) {
+                context.invalid(id,
+                        String.format("The CGMES terminal and/or the voltage level associated to the topological node %s linked to the SV injection %s is missing",
+                                tn, id));
+                return;
+            }
+            voltageLevel = context.network().getVoltageLevel(context.cgmes().voltageLevel(cgmesTerminal));
+            if (context.nodeBreaker()) {
+                node = context.nodeMapping().iidmNodeForTerminal(cgmesTerminal, voltageLevel);
+            }
+        } else {
+            voltageLevel = associatedTerminal.getVoltageLevel();
+            if (context.nodeBreaker()) {
+                VoltageLevel vl = associatedTerminal.getVoltageLevel();
+                VoltageLevel.NodeBreakerView nb = vl.getNodeBreakerView();
+                node = nb.getNodeCount();
+                nb.setNodeCount(nb.getNodeCount() + 1);
```

Comment:
The method `iidmNodeForTerminal` reserves space in the array of nodes in chunks, and keeps a counter of how many nodes have been used. Here we are skipping this check/control because we are always adding always one more node to the array. Not sure if we are just creating some holes of nodes that could be not used in the future or we could ran into some problem.

Reply:
Okay, I created another method in `NodeMapping` to keep consistency, what do you think?